### DISCUSSION TITLE
gesp-2-exam-syllabus-computer-storage/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#357] [General]
# gesp-2-exam-syllabus-computer-storage/

GESP C++二级官方考试大纲中，共有9条考点，本文针对C++（1）号知识点进行总结梳理。 （1）了解计算机存储的基本概念及分类，了解随机存储器（RAM）、只读存储器（ROM）和高速缓冲存储器（Cache）的功能及区别。

https://www.coderli.com/gesp-2-exam-syllabus-computer-storage/

**Comment by lihongzheshuai** [comment]
好！概括全面，值得学习
